src/animate.jl

```diff
@@ -1,24 +1,3 @@
-"""
-Interpolations.jl requires that one(::Type{T}) be defined for any data
```

Comment:
It looks like Interpolations.jl has fixed this issue, so this entire type is no longer required. Hooray!